pageserver/src/pgdatadir_mapping.rs

```diff
@@ -90,6 +91,24 @@ pub enum LsnForTimestamp {
     NoData(Lsn),
 }
 
+#[derive(Debug, Clone, Copy, Default)]
+pub struct LsnRange {
+    pub effective_lsn: Lsn,
+    pub request_lsn: Lsn,
+}
+
+impl LsnRange {
+    pub fn pitr(lsn: Lsn) -> LsnRange {
```

Comment:
Can you please remove the usage of the pitr acronym:
* here
* in metric names
* in comments

I'm still not sure what it means :)

Reply:
Done